backend/src/cache/server/mutation_test.go

```diff
@@ -211,13 +218,12 @@ func TestCacheNodeRestriction(t *testing.T) {
 	executionCache := &model.ExecutionCache{
 		ExecutionCacheKey: "f5fe913be7a4516ebfe1b5de29bcb35edd12ecc776b2f33f10ca19709ea3b2f0",
 		ExecutionOutput:   "testOutput",
-		ExecutionTemplate: `{"container":{"command":["echo", "Hello"],"image":"python:3.7"}},"affinity:{"nodeAffinity":{}}"`,
+		ExecutionTemplate: `{"container": {"command": ["echo", "Hello"],"image": "python:3.7"}, "nodeSelector": {"disktype": "ssd"}}`,
```

Comment:
I'm not sure the spaces should be inserted. AFAIK, the hash is calculated on JSON without these spaces.

Reply:
I removed the space but don't think that was the issue in this case.